src/data/faq.json

```diff
@@ -1125,7 +1125,8 @@
                         "title": "[Google/Android]: Error when communicating with Google API(10)",
                         "anchor": "API10",
                         "active": true,
-                        "textblock": [
+                        "textblock": [
+                            "<b>UPDATE 14.10.2020</b><br/>Should you experience this issue, we would appreciate receiving your report here: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1962' target='_blank' rel='noopener'>cwa-app-android/issues/1962</a>.<hr/>",

```

Comment:
The date should be 14.01.2021 (today), not 14.10.2020.

Reply:
thanks

Reply:
thanks